packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -1,22 +1,27 @@
 import {
     CeramicApi,
+    CreateOpts,
     Doctype,
     DoctypeConstructor,
     DoctypeStatic,
-    DocOpts,
+    LoadOpts,
+    UpdateOpts,
     UnsignedCommit,
     GenesisCommit,
 } from '@ceramicnetwork/common';
 import { AuthProvider, LinkProof } from "@ceramicnetwork/blockchain-utils-linking";
 import { CommitID, DocID, DocRef } from "@ceramicnetwork/docid";
 import { AccountID } from "caip";
 import { DID } from "dids";
-import cloneDeep from "lodash.clonedeep";
 
 const throwReadOnlyError = (): Promise<void> => {
     throw new Error('Historical document commits cannot be modified. Load the document without specifying a commit to make updates.')
 }
 
+const DEFAULT_CREATE_OPTS = { anchor: true, publish: true, sync: true }
```

Comment:
I thought about having some standard default opts exported from `docopts.ts`, but the defaults are already different between Caip10Link and TileDocument so I felt like it made sense to keep the defaults separate and specific, even if it results in some slight duplication